<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemogrCertificatesDialog</class>
 <widget class="QDialog" name="DemogrCertificatesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сервис Демография (умершие)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSyncSelected">
       <property name="text">
        <string>Синхронизировать список</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFindClient">
       <property name="text">
        <string>Найти соответствие</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnShowReport">
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectAll">
       <property name="text">
        <string>Выделить все</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearSelection">
       <property name="text">
        <string>Отменить все</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="CTableView" name="tblCertificates">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTextBrowser" name="txtCertificateInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblCertificatesCount">
     <property name="text">
      <string>Список пуст</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Фильтр</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkFilterSurname">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>14</height>
         </size>
        </property>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtFilterSurname">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkFilterName">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>14</height>
         </size>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtFilterName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkFilterPatronymic">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>14</height>
         </size>
        </property>
        <property name="text">
         <string>Отчество</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtFilterPatronymic">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Дата смерти с</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="edtDeathDateFrom"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>по</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="edtDeathDateTo"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="filterButtonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
